### ISSUE TITLE
Summarize the Bug/Issue Here

### ISSUE THREAD

**Issue by Zergiv** [surfbryce/beautiful-lyrics#1058]
### How did you install Beautiful Lyrics?

Marketplace

### What is the Bug/Issue?

Spotify gets very laggy when I use beautiful lyrics, it didn't happen to me, I had been using it for about 1 year, but I updated spotify and when I reinstalled BL and put the lyrics, it laggy the whole spotify application.

### What Country are you from?

Spain

### What Operating System are you using?

Windows

### Spotify/Spicetify Information

![image](https://github.com/user-attachments/assets/00fddd07-ecff-4d4d-ba8a-4c816b0a8af9)

### Spotify HTML/CSS and Console Screenshots

![image](https://github.com/user-attachments/assets/5c05bfdb-7699-4219-bff4-2405c81f3031)

https://file.io/PIAkmzjKjteL (HTML AND CSS) GITHUB DOESN'T LET ME UPLOAD THEM DIRECTLY

### Information Checklist

- [X] Spotify/Spicetify Information
- [X] Spotify HTML File
- [X] Spotify CSS File
- [X] Screenshots of Developer Tools Console Tab

**Comment by Magnisimus** [comment]
+1
Have same issue after the problem with BL. The most annoying thing that UI is lagging stronger if there is karaoke mode available for the song

**Comment by DaRealTurtyWurty** [comment]
Yeah I'm having the same issue. It uses up more of my cpu than even some games do. Definitely never used to be like this.

**Comment by marriedtopython** [comment]
> Yeah I'm having the same issue. It uses up more of my cpu than even some games do. Definitely never used to be like this.

Maybe its due to Spotify (using the actual app for songs and wtv) and spicerify using its skin on top of Spotify. Spotify has to reach out to its servers everytime a song is played and the spicetify theme is always running, this can use a lot of network + cpu power

**Comment by Zergiv** [comment]
> > Yeah I'm having the same issue. It uses up more of my cpu than even some games do. Definitely never used to be like this.
> 
> Maybe its due to Spotify (using the actual app for songs and wtv) and spicerify using its skin on top of Spotify. Spotify has to reach out to its servers everytime a song is played and the spicetify theme is always running, this can use a lot of network + cpu power

It should not happen suddenly, there is a bug or some flaw and it has to be fixed, nothing else.

**Comment by Zergiv** [comment]
> > > > Yeah I'm having the same issue. It uses up more of my cpu than even some games do. Definitely never used to be like this.
> > 
> > 
> > > 
> > 
> > 
> > > Maybe its due to Spotify (using the actual app for songs and wtv) and spicerify using its skin on top of Spotify. Spotify has to reach out to its servers everytime a song is played and the spicetify theme is always running, this can use a lot of network + cpu power
> > 
> > 
> > It should not happen suddenly, there is a bug or some flaw and it has to be fixed, nothing else.
> 
> Not really

Justify your answer, if the extension was working fine, and many people started to crash the extension all of a sudden, something happened, spotify updated something that made it incompatible or something changed, you don't rely on anything when you try to ''help'', if it was performance I shouldn't have any problem, I have a computer powerful enough to use spotify with an extension, besides, the lag in the UI is not the only thing that has happened recently

https://github.com/surfbryce/beautiful-lyrics/issues/1064
https://github.com/surfbryce/beautiful-lyrics/issues/1068
https://github.com/surfbryce/beautiful-lyrics/issues/1064

**Comment by marriedtopython** [comment]
> > > > > Yeah I'm having the same issue. It uses up more of my cpu than even some games do. Definitely never used to be like this.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > Maybe its due to Spotify (using the actual app for songs and wtv) and spicerify using its skin on top of Spotify. Spotify has to reach out to its servers everytime a song is played and the spicetify theme is always running, this can use a lot of network + cpu power
> 
> > > 
> 
> > > 
> 
> > > It should not happen suddenly, there is a bug or some flaw and it has to be fixed, nothing else.
> 
> > 
> 
> > Not really
> 
> 
> 
> Justify your answer, if the extension was working fine, and many people started to crash the extension all of a sudden, something happened, spotify updated something that made it incompatible or something changed, you don't rely on anything when you try to ''help'', if it was performance I shouldn't have any problem, I have a computer powerful enough to use spotify with an extension, besides, the lag in the UI is not the only thing that has happened recently
> 
> 
> 
> https://github.com/surfbryce/beautiful-lyrics/issues/1064
> 
> https://github.com/surfbryce/beautiful-lyrics/issues/1068
> 
> 

Well if you had your research, the extension broke a while ago. and recently literally 2 days ago it was fixed. I'm trying to help,  your criticising someone who tries to help people with problems. you have 0 power over me here. You're on github. Do not tell people that we don't rely on anything to help, I researched to try help YOU.

**Comment by dhafinzulkar** [comment]
> > > > > > Yeah I'm having the same issue. It uses up more of my cpu than even some games do. Definitely never used to be like this.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > Maybe its due to Spotify (using the actual app for songs and wtv) and spicerify using its skin on top of Spotify. Spotify has to reach out to its servers everytime a song is played and the spicetify theme is always running, this can use a lot of network + cpu power
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > It should not happen suddenly, there is a bug or some flaw and it has to be fixed, nothing else.
> > 
> > 
> > > 
> > 
> > 
> > > Not really
> > 
> > 
> > Justify your answer, if the extension was working fine, and many people started to crash the extension all of a sudden, something happened, spotify updated something that made it incompatible or something changed, you don't rely on anything when you try to ''help'', if it was performance I shouldn't have any problem, I have a computer powerful enough to use spotify with an extension, besides, the lag in the UI is not the only thing that has happened recently
> > #1064
> > #1068
> 
> Well if you had your research, the extension broke a while ago. and recently literally 2 days ago it was fixed. I'm trying to help, your criticising someone who tries to help people with problems. you have 0 power over me here. You're on github. Do not tell people that we don't rely on anything to help, I researched to try help YOU.

i have rtx 3070, and my spotify lag cause of beautiful lyrics

**Comment by ImAlwaysBoredlmao** [comment]
Has this issue been fixed? I'm lagging a ton.

**Comment by Zergiv** [comment]
> Has this issue been fixed? I'm lagging a ton.

Nope, I guess when they fix it, they will close the issue.

**Comment by lollilol** [comment]
> i have rtx 3070, and my spotify lag cause of beautiful lyrics

its being rendered on the CPU, even when hardware acceleration is turned on.

**Comment by ImAlwaysBoredlmao** [comment]
I figured out how to fix it. Just turn off hardware acceleration in settings.
